src/writer/serve.py

```diff
@@ -150,7 +150,8 @@ def _get_run_starter_pack(payload: InitSessionResponsePayload):
             components=payload.components,
             userFunctions=payload.userFunctions,
             extensionPaths=cached_extension_paths,
-            featureFlags=payload.featureFlags
+            featureFlags=payload.featureFlags,
+            abstractTemplates=abstract.templates
```

Comment:
**I like the idea of ​​having a single mechanism for reporting different forms of definitions interpretable by the frontend**.